spec/features_helper.rb

```diff
@@ -16,6 +16,8 @@
   Capybara::Selenium::Driver.new(app, browser: :chrome, options: options)
 end
 
+Capybara.server_port = 9887 + ENV["TEST_ENV_NUMBER"].to_i
+
```

Comment:
That's for parallel running of feature specs?

Reply:
Yep, I was getting a intermittent failure where rspec was trying to start two servers on the same port